core/src/navigation/GlobalNav.html

```diff
@@ -10,6 +10,26 @@
           class="fd-nested-list__item {node === selectedNode ? 'is-selected' : ''}"
           data-testid="{getTestId(node)}"
         >
+          {#if LuigiConfig.getConfigBooleanValue('navigation.addNavHrefs')}
+          <a href="{getRouteLink(node)}" title="{$getTranslation(node.label)}">
+            <div class="lui-fd-nested-list__content" on:click="{() => handleClick(node)}">
```

Comment:
When user clicks on a global menu. Which one handle the redirection?  The onClick event of div or the hyperlink of a tag?

Reply:
Just adding on top to check whether the `on:click ` needs to be extended with `on:click|preventDefault` to avoid full page reload on click. Remember this issue when was working on a similar task.

Reply:
From the LogoTitle.html it might be possible to move `on:click` under the `<a></a>` without breaking anything. @wdoberschuetz , please check. Thanks

Reply:
@stanleychh The ```on:click``` handles the redirection before ```href``` can do it.

@UlianaMunich Thanks, I've used your suggestions :).

Reply:
@wdoberschuetz , but do we need to have one on `div` and the other on `<a>`? I guess even without `on:click` event , only keep `href`. It is still working fine. 
@UlianaMunich , do you have any thought about this ?

Reply:
I made it this way, to be able to deduplicate the part in the ```div``` which is the same here and lower in the file as well.